handlerfunc/adapter.go

```diff
@@ -18,8 +19,18 @@ func New(handlerFunc http.HandlerFunc) *HandlerFuncAdapter {
 	}
 }
 
-func (h *HandlerFuncAdapter) Proxy(event events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
-	req, err := h.ProxyEventToHTTPRequest(event)
+// Proxy receives an API Gateway proxy event, transforms it into an http.Request
+// object, and sends it to the http.HandlerFunc for routing.
+// It returns a proxy response object gneerated from the http.ResponseWriter.
+func (h *HandlerFuncAdapter) Proxy(req events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
+	return h.ProxyWithContext(context.Background(), req)
+}
+
+// ProxyWithContext receives runtime context and an API Gateway proxy event,
```

Comment:
you shouldn't necessarily refer to `context.Context` as a "runtime context" (in general, but also because that has a specific meaning in aws-lambda-go). If you just say "context", that's understood to mean `context.Context`